src/test/java/io/cdap/plugin/google/common/GoogleAuthBaseConfigTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.google.common;
+
+import io.cdap.plugin.google.drive.source.GoogleDriveSourceConfig;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class GoogleAuthBaseConfigTest {
+
```

Comment:
can you also add tests for validation error handling?

Reply:
Added tests for validation error handling in the last update.